data/lang/module-policepatrol/en.json

```diff
@@ -51,6 +51,22 @@
     "description": "",
     "message": "You have entered the {system} system. Respect the law!"
   },
+  "RESTRICTIONS_ESTABLISHED_1": {
+    "description": "",
+    "message": "The threat has been eliminated. The restrictions are active again!"
+  },
+  "RESTRICTIONS_ESTABLISHED_2": {
+    "description": "",
+    "message": "Cease fire, cease fire!"
+  },
+  "RESTRICTIONS_WITHDRAWN_1": {
+    "description": "",
+    "message": "All restrictions withdrawn!"
```

Comment:
This is a bit confusing to the player (and to the in-universe pilot unless it's clearly established jargon/brevity code). I'd recommend something along the lines of "\<LX-1204>, you are authorized to open fire." or "You are cleared weapons free, \<LX-1204>!", replacing the brackets with the player's ship registration ID.

Similarly, "the restrictions are active again" might be better served as "Weapon authorization withdrawn. \<LX-1204>, hold fire."

Reply:
As for "the restrictions are active again", I'm thinking "weapons fire restrictions are now in effect" sound better (assuming it's not "affect", I hope?). Plus include ship label, as suggested above